pom.xml

```diff
@@ -19,7 +19,7 @@
         <dependency>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-compiler-plugin</artifactId>
-            <version>3.13.0</version>
+            <version>4.0.0-beta-1</version>
```

Comment:
Beta version means there is no guarantee that all features work stable. Please avoid of using non-stable versions if there isn't a special reason for that.